### PR TITLE
fix issue 1037

### PR DESCRIPTION
**Pull request by kamos1**
## What does this PR do?
- This pr prevents users from entering multiple newline characters and submitting a blank comment. 

## How do I test this PR?
- Press "return" multiple times and the "Post" button should remain light gray (disabled)

### CODE REVIEW

**Comment by kgardnr** [comment]
Great! Thank you @kamos1!!

**Review by kgardnr — APPROVED** [review]

File: client/talk-plugin-commentbox/CommentForm.js [hunk 1]
```diff
@@ -82,11 +82,16 @@ export class CommentForm extends React.Component {
...
+    return string.replace(/\n/g, '');
+  }
+
   render() {
     const {maxCharCount, submitEnabled, cancelButtonClassName, submitButtonClassName, charCountEnable, body, loadingState} = this.props;
 
-    const length = body.length;
-    const isRespectingMaxCount = (length) => charCountEnable && maxCharCount && length > maxCharCount;
+    const cleanedBody = this.removeNewLine(body);
+    const length = cleanedBody.length;
```
Comment (alinhle) [inline client/talk-plugin-commentbox/CommentForm.js]: Even better when we just use `trim` e.g. `const length = body.trim().length`.
That way we also prevent empty spaces, tabs and other whitespace characters.

**Review by alinhle — CHANGES_REQUESTED** [review]

**Comment by kamos1** [comment]
@cvle I added your suggestion, much cleaner.

**Review by alinhle — APPROVED** [review]
Awesome, thanks for your contribution!

**Pull request merged**